org.gnome.Devhelp.json

```diff
@@ -1,6 +1,6 @@
 {
     "app-id": "org.gnome.Devhelp",
-    "runtime": "org.gnome.Platform",
+    "runtime": "org.gnome.Sdk",
     "runtime-version": "41",
```

Comment:
42 is out, it's a good opportunity.

Reply:
I'm a bit unsure about the version mismatch (no devhelp 42 release so far afaict). It runs fine, but there may be some subtleties that I'm missing (like style glitches).

@ebassi what's your take here?

Reply:
No, I think it's fine to bump it to 42.

Reply:
Actually, it's kind of necessary, since we started building the proper docs for org.gnome.Sdk in this cycle.